### PR TITLE
INTG-236, INTG-237, INTG-242, INTG-303: Add the following to the dev …

### PR DESCRIPTION
**Pull request by dimitrikoubaroulis**
…portal: changelog, API reference, versioning, SDKs, Swagger, link to audit exporter tool.

### CODE REVIEW

File: source/includes/_swagger.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Swagger spec
+
+The up to date Swagger spec file of the SafetyCulture API is [here](https://api.safetyculture.io/docs/swagger).
```
Comment (koresar) [inline source/includes/_swagger.md]: Maybe warn about caveats it has?
btw, are there any?

File: source/includes/_reference.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# API Reference
+
+The API reference auto-generated from our [Swagger spec](#swagger-spec) can be found [here](https://api.safetyculture.io/docs).
```
Comment (koresar) [inline source/includes/_reference.md]: I'd recommend adding something like the following:
> On that page you can try calling SafetyCulture API right from your browser. Get a security token on the [SafetyCulture app Profile page](https://app.safetyculture.io/#/account/myprofile).

File: source/includes/_changelog.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+2017-04-06: Added ability to archive and un-archive audits (see [Modify audit](#modify-audit))
```
Comment (koresar) [inline source/includes/_changelog.md]: 👍

**Comment by koresar** [comment]
I'm not liking this huge list. Bad UX.
But there is nothing we can do about it right now. Will create an issue to follow this up.

image

**Comment by koresar** [comment]
Created issue: https://safetyculture.atlassian.net/browse/INTG-305

File: source/includes/_reference.md (hunk 2 above)
Comment (dimitrikoubaroulis) [inline source/includes/_reference.md]: Good idea. I like it.

File: source/includes/_swagger.md (hunk 1 above)
Comment (dimitrikoubaroulis) [inline source/includes/_swagger.md]: Don't know if any caveats. Perhaps we can mention that it does not cover the expected return status codes.

**Pull request merged**